qcengine/config.py

```diff
@@ -53,12 +53,15 @@ def get_global(key: Optional[str] = None) -> Union[str, Dict[str, Any]]:
             _global_values["cpu_brand"] = _global_values["cpuinfo"]["brand_raw"]
         except KeyError:
             # Remove this if py-cpuinfo is pinned to >=6.0.0
-            _global_values["cpu_brand"] = _global_values["cpuinfo"]["brand"]
+            try:
```

Comment:
The nested `try/except` blocks seem a bit unnecessary. Would you mind changing to a `.get('brand', 'unknown')` syntax here so we store `cpu_brand='unknown'` if the resolution fails twice?